src/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "mat-theme-generator",
+  "short_name": "mat-theme-generator",
+  "theme_color": "#1976d2",
+  "background_color": "#fafafa",
+  "display": "standalone",
+  "scope": "/",
```

Comment:
Currently app is hosted under **ng-mat-theme-generator/** so let's set scope and start_url accordingly.